devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java

```diff
@@ -117,6 +121,31 @@
     @Parameter(property = "ignoredEntries")
     private String[] ignoredEntries;
 
+    /**
+     * Custom entries into the Manifest can be configured like:
+     * <code><pre>
+     * &#x3C;configuration&#x3E;
+     *   &#x3C;manifestSections&#x3E;
+     *     &#x3C;manifestSection&#x3E;
+     *       &#x3C;name&#x3E;Custom Entries&#x3C;/name&#x3E;
+     *       &#x3C;manifestEntries&#x3E;
+     *          &#x3C;manifestEntry&#x3E;
+     *             &#x3C;entryKey&#x3E;Entry_Key1&#x3C;/entryKey&#x3E;
+     *             &#x3C;entryValue&#x3E;Entry Value 1&#x3C;/entryValue&#x3E;
+     *          &#x3C;/manifestEntry&#x3E;
+     *          &#x3C;manifestEntry&#x3E;
+     *             &#x3C;entryKey&#x3E;Entry_Key2&#x3C;/entryKey&#x3E;
+     *             &#x3C;entryValue&#x3E;Entry Value 2&#x3C;/entryValue&#x3E;
+     *          &#x3C;/manifestEntry&#x3E;
+     *       &#x3C;/manifestEntries&#x3E;
+     *     &#x3C;/manifestSection&#x3E;
+     *   &#x3C;/manifestSections&#x3E;
+     * &#x3C;/configuration&#x3E;
+     * </pre></code>
+     */
+    @Parameter(property = "manifestSections")
+    private List<ManifestSection> manifestSections;
```

Comment:
Configuration of the Mojo is really legacy at this point so there isn't much point in adding more. All Quarkus configuration is now done via the Quarkus Config System (of which `ManifestConfig` which you have already changed is a part)